lms/static/scripts/postmessage_json_rpc/server/test/index-test.js

```diff
@@ -0,0 +1,80 @@
+import { startRpcServer, getSidebarWindow, $imports } from '../index';
+
+describe('postmessage_json_rpc/server/index', () => {
+  let FakeServer;
+  let FakeRegister;
```

Comment:
```suggestion
  let fakeRegister;
```

The convention in our code is to use `FakeXYZ` if the thing you are faking has a capitalized name or `fakeXYZ` otherwise, and in this case the faked method is just called `register`.

Reply:
gotcha!